Forms/HTMLEditor/HTMLEditorField_Embed.php

```diff
@@ -106,7 +106,7 @@ public function getPreviewURL()
 		}
 
 		// Default media
-		return FRAMEWORK_DIR . '/images/default_media.png';
+		return FRAMEWORK_DIR . '/client/dist/images/src/default_media.png';
```

Comment:
It's a bit hard to tell which commit this belongs to, since that line has been changed in the PR afterwards - I'd say just leave it here, unless you can point out a commit